spec/fixtures/mocks/incomplete_tasks_with_framework_and_latest_submission.json

```diff
@@ -256,10 +252,8 @@
       "id": "67e7a34f-5d4c-4946-b045-da77f4b651db",
       "type": "submissions",
       "attributes": {
-        "framework_id": "fbe9c2dd-ad80-4398-8c92-f50a6a47d92e",
-        "supplier_id": "533a1357-4faf-4ced-b759-3e6da0bc5f3e",
-        "task_id": "855d1dd2-d9b3-4432-8c17-b63e90e50bcd",
-        "status": "in_review"
+        "status": "in_review",
+        "submitted_at": null
```

Comment:
Am I right in thinking the "submitted_at" will only have a value if/when a task has actually been successfully submitted?

Reply:
yep, that's correct.